src/NServiceBus.Core/Endpoint.cs

```diff
@@ -11,9 +11,8 @@ public static class Endpoint
         /// Creates a new initializable endpoint based on the provided configuration.
         /// </summary>
         /// <param name="configuration">Configuration.</param>
-        public static IInitializableEndpoint Prepare(EndpointConfiguration configuration)
+        static IInitializableEndpoint Prepare(EndpointConfiguration configuration)
```

Comment:
And that means you can remove `IInitializableEndpoint` as well?

Deleted code == good code